ios/FluentUI/Controls/SearchBar.swift

```diff
@@ -229,6 +239,12 @@ open class SearchBar: UIView {
 
     private var originalIsNavigationBarHidden: Bool = false
 
+    ///indicates search in progress
```

Comment:
nit space

Reply:
Fixed

---
In reply to: [469549952](https://github.com/microsoft/fluentui-apple/pull/174#discussion_r469549952) [](ancestors = 469549952)